doc/source/users_guide/setting-up-and-running-a-case/master_list_file.rst

```diff
@@ -2,6 +2,8 @@
 CTSM History Fields
 ===================
 
+CAUTION: Not all variables are relevant / present for all CTSM cases.
+
```

Comment:
Diffs in this file other this disclaimer are caused by changes that have occurred prior to this PR.